package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "cookie-restrictions-strict-list-study",
+  "name": "cookie-restrictions-strict-list",
```

Comment:
Out of curiosity, why this change?

Reply:
mythmon requested a new extension id so we can classify it as a different experiment. I don't know the details as to why.